### ISSUE TITLE
Integrate dockerfiles in docker compose

### ISSUE THREAD

**Issue by Ahsankkhan** [amosproj/amos2024ss04-building-information-enhancer#132]
## Description
* Integrate datapipeline and db dockerfiles in docker compose.

## Acceptance criteria
* When Docker compose file is executed it should also run datapipeline project and db in docker containers
* DB container should be connected with datapipeline and API project.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
